loryblu/loryblu/Feature/Locbook/View/FrequencyRotineView.swift

```diff
@@ -70,10 +70,18 @@ struct FrequencyRotineView: View {
             Spacer()
 
             LBButton(title: LBStrings.General.confirm) {
-                formConfig
-                props.onSubmit?()
+                formConfig.task.frequency = formConfig.makeFrequency()
+                Task  {
```

Comment:
>Observação: Caso o usuário não tenha clicado em nenhum turno, o valor `$formConfig.task.shift` vai ser nulo

Esse comportamento do usuário pode acontecer, pois o turno `manhã` já está como `true`, induzindo o usuário a não clicar em nenhum, já que aparentemente já está selecionado.

Portanto, o que podemos fazer é : 

## Opção 1

`if formConfig.task.shift == nil {
                    formConfig.task.shift = formConfig.period
                }`
## Opção 2
Inicializando `self.props.task.shift = .morning`
      
     init(props: Props, formConfig: FormConfig = FormConfig()) {
        var config = formConfig
        self.props = props
        self.props.task.shift = .morning
        self._formConfig = State(initialValue: config)
     }

> Observação : Inicializo o `props.task.shift` ao invés do `formConfig.task.shift`, pois o primeiro é quem vai definir o valor do `$formConfig.task`

Se você for tentar inicializar com o formConfig.task.shift = .morning, lá no método `.onAppear`vai acontecer isso : 
   

    .onAppear  {
            print(formConfig.task.shift) // resultado  = morning
            formConfig.task = props.task
            print(formConfig.task.shift) // resultado = nil
     }